lib/presentation/view/home.dart

```diff
@@ -1,15 +1,237 @@
+import '../../core/util/text_constants.dart';
+import '../controller/recipes_controller.dart';
+import '../widget/company_logo.dart';
+import '../widget/footer.dart';
+import '../widget/recipe_card.dart';
+import '../widget/recipe_card_vertical.dart';
 
 class Home extends StatelessWidget {
-  const Home({Key? key}) : super(key: key);
+  final RecipesController recipesController = Get.find<RecipesController>();
+
+  Home({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      body: Center(
-        child: Text(
-          'Home placeholder',
-        ),
+    return Scaffold(
+      backgroundColor: Palette.mainBackground,
+      body: CustomScrollView(
+        shrinkWrap: true,
+        physics: const BouncingScrollPhysics(),
+        slivers: [
+          SliverLayoutBuilder(
+            builder: (BuildContext context, SliverConstraints constraints) {
+              return SliverAppBar(
+                shadowColor: Colors.black54,
+                pinned: true,
+                backgroundColor: Palette.primary,
+                actions: [
+                  Padding(
+                    padding: EdgeInsets.symmetric(
+                      horizontal: MediaQuery.of(context).size.width *
+                          Dimensions.fivePercent,
+                    ),
+                    child: Image.asset(
+                      AssetConstants.searchIcon,
+                      width: Dimensions.searchIconWidth,
+                      color: Colors.white,
+                    ),
+                  ),
+                ],
+                title: Padding(
+                  padding: EdgeInsets.symmetric(
+                    horizontal: MediaQuery.of(context).size.width *
+                        Dimensions.fivePercent,
+                  ),
+                  child: const CompanyLogo(),
+                ),
+                expandedHeight: Dimensions.sliverAppbarExtendedHeight,
+                flexibleSpace: FlexibleSpaceBar(
+                  collapseMode: CollapseMode.parallax,
+                  centerTitle: true,
+                  title: constraints.crossAxisExtent >= Dimensions.smallScreen
+                      ? const Text(TextConstants.slogan)
+                      : null,
+                  background: Stack(
+                    fit: StackFit.expand,
+                    children: [
+                      Image.asset(
+                        AssetConstants.sliverBackground,
+                        fit: BoxFit.cover,
+                      ),
+                      const ColoredBox(
+                        color: Palette.darkOverlay,
+                      ),
+                    ],
+                  ),
+                ),
+              );
+            },
+          ),
+          SliverToBoxAdapter(
+            child: Padding(
+              padding: EdgeInsets.symmetric(
+                horizontal: (MediaQuery.of(context).size.width *
```

Comment:
You could replace MediaQuery.of(context).size.width to Get.width

Reply:
Done